src/equations/ideal_glm_mhd_2d.jl

```diff
@@ -1384,6 +1398,15 @@ end
             cons[9]^2 / 2)
 end
 
+# State validation for subcell limiting using Newton-bisection method
+@inline function is_valid_state(cons, equations::IdealGlmMhdEquations2D)
```

Comment:
@ranocha Is this name ok for you or do you have a better idea?

Reply:
This one does also not follow the naming convention we have adapted in Trixi.jl. For example, we use `have_constant_speed` instead of `equations_have_constant_speed`, `have_nonconservative_terms` instead of `equations_have_nonconservative_terms` etc. Thus, `is_valid` would be more consistent with what we have so far. Then, you could directly use the Base function `isvalid`

Reply:
I don't follow. By your argument, wouldn't a more consistent name be `have_valid_state` and thus `is_valid_state` would be very similar to that?

But I'd be ok with using `isvalid` as well. What is your preference?

Reply:
We're asking the question "Do the equations have a constant speed?", using the call `have_constant_speed(equations)`. In this case, we're asking "Is this state valid for the equations?" with the call `is_valid_state(state, equations)`. Thus, it could be just `isvalid(state, equations)` or `isvalid(u, equations)`.

Reply:
Whether the equations have a valid state should always be answered with "Yes" - otherwise, they wouldn't make any sense at all.

Reply:
OK, makes sense. @bennibolm would you please also change this to `Base.isvalid` and request my review again? With these two changes in, we can merge immediately.

Reply:
Done in [3cd1c95](https://github.com/trixi-framework/Trixi.jl/pull/1738/commits/3cd1c9508c4b79f064c4991ace89d678c99539f5).